pandas/core/base.py

```diff
@@ -942,7 +948,27 @@ def argmax(self, axis=None, skipna=True, *args, **kwargs):
 
         See Also
         --------
-        numpy.ndarray.argmax
+        numpy.ndarray.argmax : Equivalent method for numpy arrays.
+        Series.argmin : Similar method, but returning the minimum.
+
+        Examples
+        --------
+        Consider dataset containing cereal calories
+
+        >>> s = pd.Series({'Corn Flakes': 100.0, 'Almond Delight': 110.0,
+        ...               'Cinnamon Toast Crunch': 120.0, 'Cocoa Puff': 110.0})
```

Comment:
@datapythonista do we care about getting an extra space before the quote here?

Reply:
Is this what you're looking for?
```
>>> s = pd.Series({'Corn Flakes' : 100.0, 'Almond Delight' : 110.0,
...               'Cinnamon Toast Crunch' : 120.0, 'Cocoa Puff' : 110.0})
```

Reply:
i meant an extra space after the `...` on the second line, so that the single-quote in Cinnamon Toast Crunch aligned with the single-quote in Corn Flakes.  Feel free to ignore, unless @datapythonista chimes in on the policy